app/assets/javascripts/controllers/manage_project_controller.js

```diff
@@ -0,0 +1,15 @@
+angular.module("OpenHq").controller("ManageProjectController", function($scope, $routeParams, $location, Project) {
+  Project.get($routeParams.slug).then(function(project) {
+    $scope.project = project;
+  });
+
+  $scope.archiveProject = function() {
+    var c = confirm("Are you sure you want to archive this project?");
```

Comment:
would presume there will be a nice angular way of doing this and we can use our nice designed confirm page, wouldn’t worry about it for this PR though